lib/pbench/test/unit/server/test_schema.py

```diff
@@ -91,13 +97,18 @@ def test_constructor(self):
         assert y.name == "foo"
         assert y.type is ParamType.JSON
 
+        z = Parameter("foo", ParamType.JSON, required=False)
+        assert not z.required
+        assert z.name == "foo"
+        assert z.type is ParamType.JSON
+
     @pytest.mark.parametrize(
         "test",
         (
             ({"data": "yes"}, False),
-            ({"data": None}, True),
             ({"foo": "yes"}, True),
             ({"foo": None, "data": "yes"}, False),
+            ({"data": None}, True),
         ),
     )
```

Comment:
If you happen to make more changes, it would be nicer if you just reverted this change (e.g., assuming that the order of these cases is not significant).

Reply:
Apparently GitHub is optimizing our experience by showing the new line and not showing the old one (I selected a region including _both_ 😿).  Glad you figgered it out!  (Next time, I'll try to make the text more explicit....)